### PR TITLE
Billing Page UI design

### PR DESCRIPTION
**Pull request by onkar-saeloun**
## Notion card
https://www.notion.so/saeloun/Front-end-create-the-billings-page-3ad957f851b44ca7a189377cd9282d02

## Summary

- Added Design for Billing Page

## Preview

https://user-images.githubusercontent.com/104751221/173338600-7deba1e2-2956-4466-9475-cc64c01782a7.mp4

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Tested Manually

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/javascript/src/components/Profile/Billing/index.tsx [hunk 1]
```diff
@@ -1,17 +1,55 @@
...
-    <div className="py-10 pl-10 mt-4 bg-miru-gray-100 h-screen">
-      <p>Payment Page Coming up</p>
+  <>
+    <div className="flex flex-col w-4/5">
+      <Header
+        title={"Billing"}
+        subTitle={"View upcoming bill amount, due date and past bills"}
+        showButtons={false}
+      // cancelAction={()=> {}}
+      // saveAction={}
```
Comment (JijoBose) [inline app/javascript/src/components/Profile/Billing/index.tsx:15]: let's remove commented code

**Review by JijoBose — APPROVED** [review]

**Comment by supriya3105** [comment]
@onkar-saeloun Please check the CI tests that are failing.

**Review by JijoBose — CHANGES_REQUESTED** [review]

File: app/javascript/src/components/Profile/Billing/Table/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const data = [];
+
+const Table = () => (
+  <table className="min-w-full mt-1 divide-y divide-gray-200">
+    <thead>
+      <TableHeader />
+    </thead>
+
+    <tbody className="min-w-full bg-miru-gray-100  divide-y divide-gray-200">
+      {data.map((data) => (
```
[the remark this answers is not in the document]
Comment (onkar-saeloun) [reply app/javascript/src/components/Profile/Billing/Table/index.tsx:15]: @JijoBose currently we are not rendering any data on the page. We are planning to do API integration afterwords. We will pass data while doing API integration.

File: app/javascript/src/components/Profile/Billing/index.tsx (hunk 1 above)
Comment (onkar-saeloun) [reply app/javascript/src/components/Profile/Billing/index.tsx:15]: Okay.

**Review by JijoBose — APPROVED** [review]

**Pull request merged**